assets/scss/layout/_header.scss

```diff
@@ -48,4 +50,33 @@
             color: $siteBrandFontColor;
         }
     }
+    $brandHeight: $siteBrandFontSize * /* body line height */ 1.618;
+}
+
+:root
+{
+    @if ($headerLayoutFlex) {
+        /* This is an approximate calculation, it won't cover all scenarios */
+        --header-height: calc(
+            max(
+                /* menu */
+                #{$fontSize * /* .nav font size */ 0.8 * /* .menu line height */ 1.5},
+                /* brand */
+                #{$brandHeight}
+            ) + #{/* .header-wrapper top+bottom margin */ 2 * $fontSize}
+        );
+
+        /* Work-around for anchors being hidden below the header */
+        p, div, ul, form, main, footer, nav, section, h1, h2, h3, h4, h5, h6
```

Comment:
Wait, I don't think we need to select so many elements - this may cause some bugs, such as the one I found on your site:

![2020-05-18-14-03-40](https://user-images.githubusercontent.com/28993043/82179604-52bf8b80-9911-11ea-84d2-ad78b8ac2458.png)

So maybe just headings?